### PR TITLE
Updated CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by HarshitAditya27**
## What is the change? 
- I have added a step-by-step guide that is reader-friendly for beginners or first-timers in open source. This guide will help beginners to understand the whole process of opening a pull request more smoothly. I have also removed as "How to Participate?" section as it is already mentioned in the README.md file. 
## Related Issue 
- This is the pull request for issue no #14

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,23 +1,76 @@
-## How to Paritcipate?
+## How to Contribute?
```
Comment (rishabh-live) [inline CONTRIBUTING.md:7]: Also, Add How to How to Participate section. It is important as many students don't know about it yet.

**Review by rishabh-live — CHANGES_REQUESTED** [review]
REst Every thing looks great

File: CONTRIBUTING.md (hunk 1 above)
Comment (HarshitAditya27) [reply CONTRIBUTING.md:7]: Ok  👍.

Comment (HarshitAditya27) [reply CONTRIBUTING.md:7]: @rishabh-live I have added "How to Participate"

**Review by rishabh-live — APPROVED** [review]
- [x] Accepted

**Pull request merged**